src/callbacks_step/save_restart_dg.jl

```diff
@@ -326,7 +396,6 @@ function load_restart_file_on_root(mesh::Union{ParallelTreeMesh, ParallelP4estMe
             end
 
             # Read variable
-            println("Reading variables_$v ($name)...")
```

Comment:
I decided to delete this as in the serial case nothing is printed and we are usually very sparse with print statements.